### ISSUE TITLE
Show crosswalk marking icons in marking type dropdown

### ISSUE THREAD

**Issue by cbeddow** [facebook/Rapid#1259]
### Description

![image](https://github.com/facebook/Rapid/assets/10554922/19901fed-c6ae-4e02-84ce-5f6d1b5fbe84)

In this dropdown when editing a crosswalk, it would be awesome to show some graphic of the crossing type, similar to below:

![image](https://github.com/facebook/Rapid/assets/10554922/6e96c946-ad95-4a57-8c0f-11ac9e5a91d5)

I can work on making an SVG for each type if it is potentially able to be displayed next to the text there.

This would not be reflected in the actual icon of the preset.

**Comment by bhousel** [comment]
Cool - this feature did get added to iD last year, so if people find it useful we could support it in Rapid too.  
I think most of the types do have icons assigned.   
see:  https://github.com/ideditor/schema-builder/issues/56

**Comment by mxxcon** [comment]
Yes, very useful feature to have. No need to remember the given names to different markings, especially since those names are often different from the tag values and on less commonly used types.
Please add it.👍

**Comment by bhousel** [comment]
Working on this today - it would be nice to get this into a patch release this week!

The ones shipped with iD are not drawn in a way that we can use them on markers, so I'll probably just redraw them.  The are all pretty simple shapes.

<img width="734" alt="Screenshot 2023-12-27 at 1 01 23 PM" src="https://github.com/facebook/Rapid/assets/38784/9f083723-2886-496f-9077-6b86bf0bd1ec">

**Comment by bhousel** [comment]
Getting closer!  The shapes are still just a bit too dense to work at 15px map pin size.
I'll refine them a bit more.

<img width="642" alt="Screenshot 2023-12-27 at 9 24 56 PM" src="https://github.com/facebook/Rapid/assets/38784/ba57f2e6-3ef9-4ca6-a513-5c4ae5cafb31">

**Comment by bhousel** [comment]
I realized I forgot to followup on this - the new presets actually shipped in yesterday's patch release v2.2.2  🎉  

I did a lot of work to make sure they look pretty good at both the 40px preset icon size and the 15px map pin size.
see: https://rapideditor.github.io/temaki/docs/#group=crossings

The hope is that users scrolling around the map should be able to tell which crossings are mapped with `crossing:markings` detail. 

I'll leave this issue open for now, because I haven't yet updated the dropdown component to show inline icons yet.
